autoload/go/debug.vim

```diff
@@ -894,8 +894,13 @@ sign define godebugbreakpoint text=> texthl=GoDebugBreakpoint
 sign define godebugcurline text== linehl=GoDebugCurrent texthl=GoDebugCurrent
 
 fun! s:hi()
-  hi GoDebugBreakpoint term=standout ctermbg=117 ctermfg=0 guibg=#BAD4F5  guifg=Black
-  hi GoDebugCurrent    term=reverse  ctermbg=12  ctermfg=7 guibg=DarkBlue guifg=White
+
+  if execute("hi GoDebugBreakpoint", "silent!") =~ "xxx cleared"
```

Comment:
`execute()` was added in Vim 7.4.2008, and we support Vim 7.4.1689 and newer, so that can't be used. I did start an issue just now to discus relaxing that requirement though: #1847

Reply:
FWIW, if the original code is just moved to `syntax`, then there won't be any need to guard the highlighting, because of the order in which highlight groups are applied.